jupyter_server/event_schemas/gateway/client/v1.yaml

```diff
@@ -0,0 +1,34 @@
+"$id": https://events.jupyter.org/jupyter_server/gateway/client/v1
+version: 1
+title: Enterprise Gateway Client activities.
+personal-data: true
+description: |
+  Record events of a gateway client.
+type: object
+required:
+ - status
+ - message
+properties:
+  status:
+    enum:
+      - error
+      - success
+    description: |
+      Status received by Gateway client upon calling EG server.
+
+      This is a required field.
+
+      Possible values:
+
+      1. error
+         Error response while calling gateway kernel.
+
+      2. success
+         Success response while calling gateway kernel.
+  status_code:
+    type: number
+    description: |
+      Http response codes based on the rest api call to gateway kernel.
+      Examples: 200, 400, 502, 503, 599 etc
+  message:
+    type: string
```

Comment:
Should this include the REST method and endpoint?

Reply:
_message_ includes exception details in case of errored scenario and a generic message from method in case of success..

Reply:
But can the event's consumer discern the endpoint that produced the event (success or failure)?  Just wondering if `endpoint` should be its own field.

Reply:
We added `gateway_url` to handle this specific scenario. Thanks!